Components/Dialog/Dialog.css

```diff
@@ -0,0 +1,102 @@
+.__dialog-bunch{
+    
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+}
+
+.__dialog {
+    display: flex;
+    flex-flow: column;
+    /* position: fixed; */
+    z-index: 10;
+    height: 25rem;
+    width: 25rem;
+    border-radius: 6px;
+    box-shadow: 0 0 25px 0 rgba(0, 0, 0, 0.25);
+    justify-content: space-between;
+    background-color: white;
+    /* transform: translate(-50%,-50%); */
+    max-height: 360px;
+    min-width: 240px;
+    max-width: 240px;
+    margin: 1rem;
+    /* display: none; */
+}
+
+.__alert-dialog{
+    height: 15rem;
+}
+
+.__dailog-header{
+    padding: 1rem;
+    font-weight: 800;
+}
+
+.__dailog-body {
+    font-weight: 600;
+    font-size: 1.5rem;
+    padding: 1rem;
+    /* max-height: 15rem;
+    overflow:; */
+}
+.__confirmation-body{
+    max-height: 15rem;
+    overflow-y:scroll;
+}
+.__dailog-body li{
+    /* padding: 5px; */
+    display: flex;
+    flex-flow: row;
+    align-items: center;justify-content: center;
+}
+
+.__dailog-body span{
+    background-color: transparent;
+}
+.__dialog-body-item{
+    padding: 7px;
+   /* display: flex; */
+   /* width: 80%; */
+   /* justify-content: center;
+   align-self: center; */
+   align-self: center;
+   flex-grow: 1;
+}
+
+
+.__dailog-footer{
+    display: flex;
```

Comment:
Any reason for the dangling classnames?

Reply:
No such reason I was having issues to did temporary changes but forgot to revert it back and committed it ,
now have undone the changes